### PR TITLE
Feature/cplp 720/app detail page

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

File: cx-portal/public/testdata/appDetails.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (oyo) [inline cx-portal/public/testdata/appDetails.json:1]: this is not used any more and can be deleted - the test data is now in a different repo
https://github.com/catenax-ng/product-portal-common-assets/blob/main/public/assets/api/apps/demo/details.json
and available at URL:
https://portal-dev.demo.catena-x.net/assets/api/apps/demo/details.json

File: cx-portal/src/components/pages/AppDetail/components/AppDetailHeader/index.tsx [hunk 2]
```diff
@@ -11,26 +12,37 @@ export default function AppDetailHeader({ item }: AppDetailHeaderProps) {
...
-        </p>
-        <Button>{t('content.appdetail.subscribe')}</Button>
+    <div className="container">
+      <div className="row">
+        <div className="lead-image">
+          <img src={item.leadPictureUri} alt={item.name} />
+        </div>
+        <div className="content">
+          <p className="provider">{item.provider}</p>
+          <h2 className="heading">{item.name}</h2>
```
Comment (oyo) [inline cx-portal/src/components/pages/AppDetail/components/AppDetailHeader/index.tsx:22]: This was from my original example where I didn't care much about the look. In the final component we should use our shared `<Typography />` for all `<p>` or `<h*>` tags

File: cx-portal/src/components/pages/AppDetail/components/AppDetailHeader/index.tsx [hunk 3]
```diff
@@ -11,26 +12,37 @@ export default function AppDetailHeader({ item }: AppDetailHeaderProps) {
...
+          <div className="rating">
+            <Rating defaultRating={item.rating} />
+            <span className="rating-number">{item.rating}</span>
+          </div>
+          <p className="price">{item.price}</p>
+          <p className="usecase">
+            <b>{t('content.appdetail.usecase')}:</b>{' '}
```
Comment (oyo) [inline cx-portal/src/components/pages/AppDetail/components/AppDetailHeader/index.tsx:29]: please don't use `<b>` tags. This is styling and should be handled in the (s)css

File: cx-portal/src/components/pages/AppDetail/index.tsx [hunk 4]
```diff
@@ -1,31 +1,79 @@
...
+            Far far away, behind the word mountains, far from the countries
+            Vokalia and Consonantia, there live the blind texts. Separated they
+            live in Bookmarksgrove right at the coast of the Semantics, a large
+            language ocean. A small river named Duden flows by their place and
+            supplies it with the necessary regelialia. It is a paradisematic
+            country, in which roasted parts of sentences fly into your mouth.
+            Even the all-powerful Pointing has no control about the blind texts
+            it is an almost unorthographic life One day however a small line of
+            blind text by the name of Lorem Ipsum decided to leave for the far
+            World of Grammar...
```
Comment (oyo) [inline cx-portal/src/components/pages/AppDetail/index.tsx:55]: this is the app long description and should be replaced with

`{ta('${item.id}.description')}`

where ta is defined at the function entry 

`const { ta } = useTranslation('apps')`

we will later provide the translations for those texts either through an api or a generated static JSON file. Doing that will require to create a little shared component `ReadMoreText` that will check and truncate the text if too long. There are many components for that - this looks easy to use: https://www.npmjs.com/package/react-show-more

**Review by oyo — CHANGES_REQUESTED** [review]
see some comments

**Review by oyo — APPROVED** [review]
looks good to me now :)

**Comment by oyo** [comment]
@jjeroch removed the rating now

**Comment by jjeroch** [comment]
> @jjeroch removed the rating now

Thanks Martin & Nidhi

**Review by jjeroch — APPROVED** [review]

**Pull request merged**